content/applications/sales/sales/send_quotations/pdf_quote_builder.rst

```diff
@@ -0,0 +1,203 @@
+    cannot be modified.
+
+Then, in the :guilabel:`Sales` section, in the :guilabel:`Visible at` field, click the drop-down
+menu, and select either: :guilabel:`Quotation`, :guilabel:`Confirmed order`, or :guilabel:`Inside
+quote`.
+
+- :guilabel:`Quotation`: the document is sent to (and accessible by) customers at any time.
+
+- :guilabel:`Confirmed order`: the document is sent to customers upon the confirmation of an order.
+  This is best for user manuals and other supplemental documents.
+
+- :guilabel:`Inside quote`: the document is included in the PDF of the quotation, between the header
+  pages and the quote table.
+
+.. example::
+   How the document link appears inside a quotation.
+
+    .. image:: pdf_quote_builder/pdf-inside-quote-sample.png
+       :align: center
+       :alt: Sample of an uploaded pdf with the inside quote option chosen in Odoo Sales.
+
+Lastly, in the :guilabel:`E-Commerce` section, decide whether or not to :guilabel:`Show on product
+page` on the front-end (in the online store).
+
+.. example::
+   How the document link appears on a typical product page in the online store.
+
+    .. image:: pdf_quote_builder/show-product-page.png
+       :align: center
+       :alt: Showing a link to an uploaded document on a product page using Odoo Sales.
+
+Dyanmic text values (product-specific)
+--------------------------------------
+
+While creating custom PDFs for quotes, using any standard PDF editor software, certain dynamic texts
+should be utilized throughout the design of the PDF to ensure Odoo can automatically fill in the
+appropriate content accordingly.
+
+Dynamic text values are, essentially, coded placeholders that Odoo fills in automatically, according
+to the specific information related to the quote.
+
+Below are the dynamic text values and what they will represent in the custom PDFs:
+
+- :guilabel:`name`: Order Reference
+- :guilabel:`partner_id_name`: Customer Name
+- :guilabel:`user_id_name`: Salesperson Name
+- :guilabel:`amount_untaxed`: Untaxed Amount
+- :guilabel:`amount_total`: Total Amount
+- :guilabel:`delivery_date`: Delivery Date
+- :guilabel:`validity_date`: Expiration Date
+- :guilabel:`client_order_ref`: Customer Reference
+
+Product-specific dynamic text values are as follows:
+
+- :guilabel:`description`: Product Description
+- :guilabel:`quantity`: Quantity
+- :guilabel:`uom`: Unit of Measure (UoM)
+- :guilabel:`price_unit`: Price Unit
+- :guilabel:`discount`: Discount
+- :guilabel:`product_sale_price`: Product List Price
+- :guilabel:`taxes`: Taxes name joined by a comma (`,`)
+- :guilabel:`tax_excl_price`: Tax Excluded Price
+- :guilabel:`tax_incl_price`: Tax Included Price
+
+PDF quote
+=========
+
+When a quotation with a configured PDF-related quote has been confirmed, click the :guilabel:`⚙️
+(gear)` icon to reveal a drop-down menu. From this drop-down menu, select :guilabel:`Print`, then
+select :guilabel:`PDF Quote`.
```

Comment:
I'm slightly confused by this sentence. This is how I think I'm following what you're saying, and let me know if my interpretation is incorrect:

After using the custom PDF builder to personalize a quotation, *and then* we sent the quote to the customer, *and then* the customer placed the order through the Customer Portal...
you can print the PDF quote with these following steps?

I'm not following why this is really useful? Could you elaborate a little more? (:

Reply:
You bring up a great point. And, frankly, I agree -- I'm not certain how this is useful. To be clear, I only added this at the bottom because it was mentioned by JCS in the initial Project Task requesting this doc to be made, so I felt obligated to throw it in towards the end.

So, given that JCS made this mention directly, I think I'll opt to keep it in -- however, I'll be sure to clean up the language, and add more of a "reasoning" behind a user (potentially) needing/wanting to do this.

Thanks for pointing this out!